src/schemas/json/typst.json

```diff
@@ -0,0 +1,245 @@
+            "type": "string"
+          },
+          "uniqueItems": true,
+          "examples": [
+            ["chart", "plot", "data visualization"],
+            ["ieee", "template", "paper"]
+          ]
+        },
+        "categories": {
+          "type": "array",
+          "description": "An array with up to three categories from the official list to help users discover the package.",
+          "items": {
+            "type": "string",
+            "enum": [
+              "components",
+              "visualization",
+              "model",
+              "layout",
+              "text",
+              "languages",
+              "scripting",
+              "integration",
+              "utility",
+              "fun",
+              "book",
+              "report",
+              "paper",
+              "thesis",
+              "poster",
+              "flyer",
+              "presentation",
+              "cv",
+              "office"
+            ]
+          },
+          "maxItems": 3,
+          "uniqueItems": true,
+          "examples": [
+            ["visualization", "components"],
+            ["paper", "layout"]
+          ]
+        },
+        "disciplines": {
+          "type": "array",
+          "description": "An array of disciplines from the official list defining the target audience. Should be empty if generally applicable.",
+          "items": {
+            "type": "string",
+            "enum": [
+              "agriculture",
+              "anthropology",
+              "archaeology",
+              "architecture",
+              "biology",
+              "business",
+              "chemistry",
+              "communication",
+              "computer-science",
+              "design",
+              "drawing",
+              "economics",
+              "education",
+              "engineering",
+              "fashion",
+              "film",
+              "geography",
+              "geology",
+              "history",
+              "journalism",
+              "law",
+              "linguistics",
+              "literature",
+              "mathematics",
+              "medicine",
+              "music",
+              "painting",
+              "philosophy",
+              "photography",
+              "physics",
+              "politics",
+              "psychology",
+              "sociology",
+              "theater",
+              "theology",
+              "transportation"
+            ]
+          },
+          "uniqueItems": true,
+          "examples": [["computer-science", "mathematics"], ["physics"]]
+        },
+        "compiler": {
+          "type": "string",
+          "description": "The minimum Typst compiler version required for this package to work (SemVer format).",
+          "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$",
```

Comment:
Since Typst 0.11 (or 0.12, I'm not sure) it is possible to leave out some components of the required compiler version: `0` or `0.13` are both valid values. And here too, pre-release tags are not supported.